src/main/java/spireMapOverhaul/SpireAnniversary6Mod.java

```diff
@@ -217,6 +221,14 @@ public void receivePostInitialize() {
         TextCodeInterpreter.addAccessible(ZoneShapeMaker.class);
     }
 
+    public static void addMonsters() {
+        for (AbstractZone zone : allZones) {
```

Comment:
I think we should work with unfilteredAllZones here, right? Seems rpoblematic not to load encounters from zones that are currently deactiveated in case they get activated again

Reply:
Yeah, I just overlooked that we'd merged in the filtering.